pylint/checkers/base/name_checker/checker.py

```diff
@@ -303,9 +303,7 @@ def _create_naming_rules(self) -> tuple[dict[str, Pattern[str]], dict[str, str]]
                 regexps[name_type] = DEFAULT_PATTERNS[name_type]
 
             custom_regex_setting_name = f"{name_type}_rgx"
-            custom_regex = getattr(
-                self.linter.namespace, custom_regex_setting_name, None
-            )
+            custom_regex = getattr(self.linter.config, custom_regex_setting_name, None)
```

Comment:
For this one I'm wondering if an hard coded value would not be better/clearer than using getattr(). It's not everyday that we add a new regex setting name. On the other hand it does permit to have a setting for a custom name pattern created by a plugin...

Reply:
Yeah, I think the current system is fine. We would just copy 10 relatively similar names.